README.md

```diff
@@ -17,11 +17,17 @@ Pubkey Encrypt leverages the Key & Encrypt modules for integrating it's encrypti
 3) [Head over to the architecture document for a detailed technical discussion on how this module works](documentation/ArchitectureDoc.pdf).
 
 ## Installation & Basic Usage
-1) Install & enable this module along with all its dependencies i.e. [Key](https://www.drupal.org/project/key), [Encrypt](https://www.drupal.org/project/encrypt) and [Encrypt PHPSecLib Encryption](https://www.drupal.org/project/encrypt_seclib) (Please note that Encrypt PHPSecLib Encryption needs PHPSecLib library installed via composer. See the [module README file](http://cgit.drupalcode.org/encrypt_seclib/tree/README.md) first before installing it).
+1) Enable this module along with all its dependencies i.e. [Key](https://www.drupal.org/project/key), [Encrypt](https://www.drupal.org/project/encrypt) and [Encrypt PHPSecLib Encryption](https://www.drupal.org/project/encrypt_seclib) (Please note that Encrypt PHPSecLib Encryption needs PHPSecLib library installed via composer. See the [module README file](http://cgit.drupalcode.org/encrypt_seclib/tree/README.md) first before installing it).
 
-2) Initialize this module from Administration >> Configuration >> System >> Pubkey Encrypt settings.
+3) Enable at least one of the modules providing an "Asymmetric Keys Generator" plugin.
```

Comment:
I think (2) is now missing.  In any case, as this is markdown, you can simply have every point be "1. ", and they will automatically get numbered accordingly when rendered.  To preserve the numbering when not rendered, you can still do "1. ", "2. ", etc., but "1) " syntax is not markdown.